HLTrigger/JetMET/plugins/HLTHFJetCleaner.cc

```diff
@@ -0,0 +1,269 @@
+// -*- C++ -*-
+//
+// Package:    HLTrigger/HLTHFJetCleaner
+// Class:      HLTHFJetCleaner
+//
+/**\class HLTHFJetCleaner HLTHFJetCleaner.cc HLTrigger/HLTHFJetCleaner/plugins/HLTHFJetCleaner.cc
+
+ Description: [one line class summary]
```

Comment:
Please provide a more detailed comment what this plugin does with the input collections etc. and what the decision is based on.

Reply:
Implemented in [e4f06dd](https://github.com/cms-sw/cmssw/pull/37539/commits/e4f06dd8ceabde9707fda329e838c4b2bbae193c)